src/Descriptor.cpp

```diff
@@ -105,10 +110,12 @@ bool Descriptor::flush_output() noexcept {
     if (outbuf_.empty())
         return true;
 
-    if (snoop_by && character) {
-        snoop_by->write(character->name);
-        snoop_by->write("> ");
-        snoop_by->write(outbuf_);
+    if (character) {
+        for (auto *snooper : snoop_by_) {
+            snooper->write(character->name);
```

Comment:
Occurs to me I can use the new for line iterator to prefix each _line_ with `char> ` instead of just the first bit. Might make it clearer. Not that we use snoop a whole lot!

Reply:
Currently it's a 95+ command. I don't think it should be used casually except in testing. If the mud ends up being played by more people then it should only be used if it looks like someone's obviously breaking rules and you can't tell by standing there while wizinvis. And perhaps then only after they've already been warned. Or if another player has complained about their behavior.